presto-spi/pom.xml

```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <scope>test</scope>
+            <scope>provided</scope>
```

Comment:
@jha-gunjan Hmm.... SPI cannot have a dependency on guava. I guess, that means that we can't use ImmutableMap in TuppleDomain. Let's find another way.

Reply:
@sopel39 @findepi Karol, Piotr, would you have a suggestion on how to make TuppleDomain produce predictable order of columns?

Reply:
@mbasmanova @jha-gunjan  Maria, Gunjan, this is already fixed in upstream -- http://github.com/prestosql/presto
🙂

Reply:
You can make something similar to https://github.com/prestosql/presto/pull/226 (or simply cherry pick that).